server/src/main/java/com/tencent/rss/server/LocalStorageChecker.java

```diff
@@ -107,6 +110,7 @@ long getUsedSpace(File file) {
 
     private final File storageDir;
     private boolean isHealthy;
+    private long usedSize;
```

Comment:
it can be named as usedSpace, and be set in getUsedSpace, no extra getXXX should be added

Reply:
`getUsedSpace` would calculate the used space of partition, we can't resue the result if rename and use getUseSpace.